internal/cli/auth/login.go

```diff
@@ -189,6 +189,14 @@ func (opts *LoginOpts) setUpProfile(ctx context.Context) error {
 
 	// Only make references to profile if user was asked about org or projects
 	if opts.AskedOrgsOrProjects && opts.ProjectID != "" && opts.OrgID != "" {
+		if !opts.ProjectExists(config.ProjectID()) {
+			return fmt.Errorf("you don't have access to this project ID or it doesn't exist")
```

Comment:
[q] @blva can these be made sentinel errors?

Reply:
more info in https://go.dev/blog/go1.13-errors

Reply:
nice! i keep forgetting :) thanks